board/bananapi/bananapi-m2-zero/rootfs_overlay/etc/init.d/S10modules

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/sbin/modprobe brcmfmac
+sleep 2
```

Comment:
Is there an issue if you remove the sleep?

Reply:
Without the sleep the wifi module would not load fast enough for the Network init script. So the wifi network would fail on boot.